docs/content/codeformat.fsx

```diff
@@ -24,7 +24,7 @@ can be called to format snippets repeatedly:
 *)
 
 let fsharpCompiler = Assembly.Load("FSharp.Compiler")
-let formattingAgent = CodeFormat.CreateAgent(fsharpCompiler)
+let formattingAgent = CodeFormat.CreateAgent()
```

Comment:
I didnt removed the parameter from the method declaration when I looked at all scrpts to see if I need to change something I saw error on this line so I change it